DigitalOffice/Services/CompanyService/Broker/Consumers/GetUserPositionConsumer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Threading.Tasks;
+using Broker.Requests;
+using Kernel.Broker;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Mappers.Interfaces;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using MassTransit;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LT.DigitalOffice.CompanyService.Broker.Consumers
+{
+    public class GetUserPositionConsumer : IConsumer<InternalGetUserPositionRequest>
+    {
+        private readonly IMapper<DbPosition, object> mapper;
+        private readonly IPositionRepository repository;
+
+        public GetUserPositionConsumer(
+            [FromServices] IPositionRepository repository,
+            [FromServices] IMapper<DbPosition, object> mapper)
+        {
+            this.repository = repository;
+            this.mapper = mapper;
+        }
+
+        public async Task Consume(ConsumeContext<InternalGetUserPositionRequest> context)
+        {
+            var response = OperationResultWrapper.CreateResponse(GetUserPosition, context.Message);
+
+            await context.RespondAsync(response);
```

Comment:
Use IOperationResult

Reply:
I call OperationResultWrapper.CreateResponse, which wraps the function result in IOperationResult on 27 line

Reply:
If "response" is of IOperationResult type (or something like that), then everything's ok.
If response is an anonymous object or just an Object object, then MassTransit will throw an exception. So please check it. I'd appreciate the working screenshots.

Reply:
U are right. I added type for response.

Reply:
@mekk1t  Screenshot attached below